### ISSUE TITLE
Training strategies of the three models provided

### ISSUE THREAD

**Issue by tqyunwuxin** [hzwer/arXiv2020-RIFE#79]
do you train the RIFE  RIFE2F  RIFE_HD using the same training strategies such as learning_rate、optimizer、training epoch and so on?

**Comment by hzwer** [comment]
Hello, we train RIFE and RIFE2F using the same training strategies. RIFE_HD is designed for better visual effect on HD videos, so I add scale augmentation for Vimeo90K. (1.5x 2x 4x 6x randomly)

**Comment by tqyunwuxin** [comment]
> Hello, we train RIFE and RIFE2F using the same training strategies. RIFE_HD is designed for better visual effect on HD videos, so I add scale augmentation for Vimeo90K. (1.5x 2x 4x 6x randomly)
if the model input size is A, do you mean 1.5xA 2x A 4xA 6xA randomly crop the data, and then resize them to A?

**Comment by tqyunwuxin** [comment]
> Hello, we train RIFE and RIFE2F using the same training strategies. RIFE_HD is designed for better visual effect on HD videos, so I add scale augmentation for Vimeo90K. (1.5x 2x 4x 6x randomly)

if the model input size is A, do you mean 1.5xA 2x A 4xA 6xA randomly crop the data, and then resize them to A?

**Comment by hzwer** [comment]
Yes. You should rescale and multiply the optical flow label as well.

**Comment by tqyunwuxin** [comment]
> Yes. You should rescale and multiply the optical flow label as well.

hello, the size of the image of vimeo90 is 256*448, the crop size that you set in project is 224*224(model input size A).  according to the above discussion，how to crop patch of size 6xA.  do you use smaller A? such as 42x42?

**Comment by tqyunwuxin** [comment]
> Yes. You should rescale and multiply the optical flow label as well.

hello, the size of the image of vimeo90 is 256x448, the crop size that you set in project is 224x224(model input size A). according to the above discussion，how to crop patch of size 6xA. do you use smaller A? such as 42x42?

**Comment by hzwer** [comment]
Resize first and then crop.
